SimPEG/utils/pgi_utils.py

```diff
@@ -151,7 +160,7 @@ def make_SimplePGI_regularization(
             alpha_y=alph_y[i],
             alpha_z=alph_z[i],
             cell_weights=clwhtlst[i],
-            **kwargs
+            **kwargs,
```

Comment:
Could be an issue here if people try to input alpha_s as kwargs. Is there a reason it should always be zero and never set? We should probably assert it in the setter